libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -507,6 +507,7 @@ if(LIBC_TYPES_HAS_FLOAT16)
     libc.src.math.canonicalizef16
     libc.src.math.ceilf16
     libc.src.math.copysignf16
+    libc.src.math.f16div
     libc.src.math.f16divf
```

Comment:
I think I could enable `f16divl` and `f16divf128` on AArch64 actually. The Buildbot failures after #96711 seemed to be caused by Clang 11 failing to round the result of `float128` arithmetic to `float16`, but FPUtil/generic/div.h doesn't have a line like FPUtil/generic/FMA.h's `return static_cast<OutType>(x * y + z);`.